package.json

```diff
@@ -76,7 +76,8 @@
       "/node_modules/(?!(@edx/paragon)/).*/"
     ],
     "globals": {
-      "courseContext": {}
+      "courseContext": {},
```

Comment:
I'm sort of wondering if instead of `courseContext`/`zendeskContext`, we could just have a standard context. Perhaps `studioContext` and this could be the course or zendesk depending on which page is grabbing the data.

This might need some upstream changes in the platform though, but it might make @MichaelRoytman's work for getting studio passing over the zendesk info a bit easier as he could reuse the existing context inserting function that @efischer19 created initially.

Reply:
Totally and could help with scaling. Are you thinking of something like `studioContext: { courseContext: {...}, zendeskContext: {...}}` or something where the data that's currently within `courseContext` would be in this new `studioContext`?

Reply:
I think it `studioContext` could just replace what's in `courseContext` -- i.e. the structure of the content is left entirely up to the page. Just an idea! I think it would be more flexible.

Reply:
Got it! I'll take a look and make sure that changing that doesn't mess with other assumptions made elsewhere in the repo.